### PR TITLE
Documentation for configuring Okta

### PR DESCRIPTION
**Pull request by chrsmith**
Documentation for configuring Okta to be used as a SAML SSO identity provider for the Pulumi Cloud Console. The service today doesn't have a smooth self-serve experience, but once the organization has been set up, this guide should walk through the necessary steps on the Okta side to get everything working together.

image

I'm not sure what I'm doing wrong with the markdown table syntax, but it is definitely not rendering the way I would have expected...

image

image

image

image

image

### CODE REVIEW

File: reference/service/saml-okta.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+![View Setup Instructions](../../images/reference/service/saml-okta/view-setup-instructions.png)
+
+Next, scroll to the bottom of the setup instructions and select the value in the large text box
+with the heading "Provide the following IDP metadata to your SP provider". That's the full SAML
+Identity Provider SSO descriptor, which contains all of the settings Pulumi needs to verify
+a user's identity.
+
+![SAML Application Metadata](../../images/reference/service/saml-okta/okta-xml-descriptor.png)
+
+With the large block of XML text in your clipboard, navigate to your SAML organization's page on
```
Comment (lukehoban) [inline reference/service/saml-okta.md]: Remove "large" here and in next paragraph.

File: reference/service/saml-okta.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+### Configuring the Application
+
+Next you will be guided through a wizard to configure the Okta application. The first step is to
+give it a name, e.g. "Pulumi Cloud Console", and an icon.
+
+![Configuring a SAML Integration](../../images/reference/service/saml-okta/create-saml-integration.png)
+
+The next step is where you configure the SAML application's settings. The values to provide will
+depend on the name of your Pulumi organization, e.g. `robot-co` or `acme`.
```
Comment (lukehoban) [inline reference/service/saml-okta.md]: Should we (for now) include an explicit note to "Contact us to provision a SAML-backed Pulumi organization to associate with you Okta SSO"?

File: reference/service/saml-okta.md (hunk 1 above)
Comment (chrsmith) [reply reference/service/saml-okta.md]: Done.

File: reference/service/saml-okta.md (hunk 2 above)
Comment (chrsmith) [reply reference/service/saml-okta.md]: Sounds reasonable.

File: reference/service/saml-okta.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+title: SAML Configuration > Okta
+---
+
+This guide walks through how to configure Okta as a SAML SSO identity provider (IdP) for use with
+the Pulumi Cloud Console.
+
+> The screen shots below are using the Okta _Classic UI_. You can switch to it by clicking the gear
```
Comment (joeduffy) [inline reference/service/saml-okta.md]: Can this not be done using the default UI?

File: reference/service/saml-okta.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+The first step is to create a new Okta Application Integration. Of the various "sign on methods"
+available, choose "SAML 2.0".
+
+![Creating an Okta Application](../../images/reference/service/saml-okta/create-okta-application.png)
+
+### Configuring the Application
+
+Next you will be guided through a wizard to configure the Okta application. The first step is to
+give it a name, e.g. "Pulumi Cloud Console", and an icon.
```
Comment (joeduffy) [inline reference/service/saml-okta.md]: Nit, would be great if "icon" linked to our icon.

File: reference/service/saml-okta.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+![Configuring a SAML Integration](../../images/reference/service/saml-okta/create-saml-integration.png)
+
+The next step is where you configure the SAML application's settings. The values to provide will
+depend on the name of your Pulumi organization, e.g. `robot-co` or `acme`.
+
+> [Contact us](https://www.pulumi.com/about/#contact-us) to provision the SAML-backed Pulumi
```
Comment (joeduffy) [inline reference/service/saml-okta.md]: Feels like this should be more prominent, in fact. It should be clearer that users cannot actually self-provision SAML organizations today and that this document is meant for users who have already done that.

File: reference/service/saml-okta.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+> organization for you to associate with Okta.
+
+<style>
```
Comment (joeduffy) [inline reference/service/saml-okta.md]: Why are you overriding the styles? Let's just use the defaults for the site.

File: reference/service/saml-okta.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+![Pulumi Organization Settings](../../images/reference/service/saml-okta/pulumi-org-settings.png)
+
+Once the IDP metadata descriptor has been saved, you are all set to log into to Pulumi.
+
+### Logging in
+
+To log into the Pulumi Cloud Console using your Okta-based credentials, navigate to a SAML-specific
+login page for your organization. For example:
+
+https://app.pulumi.com/welcome/robot-co/sso
```
Comment (joeduffy) [inline reference/service/saml-okta.md]: Feels like it's worth a note saying that the default app.pulumi.com login won't work.

**Pull request merged**